cmd/list.go

```diff
@@ -53,23 +55,43 @@ that you did in the past. Examples:
 
 The date format is that of YYYY-MM-DD for getting specific dates.`,
 	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println("list called")
 		var date []byte
 		if len(args) == 0 {
 			date = cfg.PreviousDate()
 		} else {
 			if args[0] == "today" {
 				date = cfg.CurrentDate()
 			}
+			// check for date and assign to a byte slice
+			if len(date) == 0 {
```

Comment:
Just roll this into an `else {` from above. If it's got an args[0] and it's not `today` it should be a date.